zerver/management/commands/update_stream_active_status.py

```diff
@@ -0,0 +1,22 @@
+import logging
+from typing import Any
+
+from django.conf import settings
+from typing_extensions import override
+
+from zerver.lib.logging_util import log_to_file
+from zerver.lib.management import ZulipBaseCommand
+from zerver.lib.streams import check_update_all_streams_active_status
+
+## Logging setup ##
+logger = logging.getLogger(__name__)
+log_to_file(logger, settings.DIGEST_LOG_PATH)
+
+
+class Command(ZulipBaseCommand):
+    help = """For all streams, update the `is_recently_active` field to False if no message has been sent to the stream in the last 180 days."""
+
+    @override
+    def handle(self, *args: Any, **options: Any) -> None:
+        count = check_update_all_streams_active_status()
+        logger.info("Marked %s streams as not recently active.", count)
```

Comment:
Did some renaming to improve comments and avoid having "channel" in user-facing strings (not bothering with function names, which are easy to change without migrations). Most significant is the `cron` job name, since those can be annoying to change without generating duplicates.

``` diff
diff --git a/puppet/zulip/manifests/app_frontend_once.pp b/puppet/zulip/manifests/app_frontend_once.pp
index 04044cf0a2..d35f3cb4ff 100644
--- a/puppet/zulip/manifests/app_frontend_once.pp
+++ b/puppet/zulip/manifests/app_frontend_once.pp
@@ -36,10 +36,10 @@ class zulip::app_frontend_once {
   }
 
   # Daily
-  zulip::cron { 'update-stream-active-status':
+  zulip::cron { 'update-channel-recently-active-status':
     hour   => '1',
     minute => '40',
-    manage => 'update_stream_active_status',
+    manage => 'update_channel_recently_active_status',
   }
   zulip::cron { 'soft-deactivate-users':
     hour   => '5',
diff --git a/zerver/management/commands/update_channel_recently_active_status.py b/zerver/management/commands/update_channel_recently_active_status.py
index 98fd973451..089748d1ff 100644
--- a/zerver/management/commands/update_channel_recently_active_status.py
+++ b/zerver/management/commands/update_channel_recently_active_status.py
@@ -14,9 +14,9 @@ log_to_file(logger, settings.DIGEST_LOG_PATH)
 
 
 class Command(ZulipBaseCommand):
-    help = """For all streams, update the `is_recently_active` field to False if no message has been sent to the stream in the last 180 days."""
+    help = """Update the `Stream.is_recently_active` field to False for channels whose message history has aged to the point where it is no longer recently active."""
 
     @override
     def handle(self, *args: Any, **options: Any) -> None:
         count = check_update_all_streams_active_status()
-        logger.info("Marked %s streams as not recently active.", count)
+        logger.info("Marked %s channels as not recently active.", count)
```